content/docs/about/policies/_index.md

```diff
@@ -37,17 +37,68 @@ Review the policies that you can configure in kgateway and the level at which yo
 | [Local rate limiting](/docs/security/local-ratelimit/) | TrafficPolicy | 
 | [Transformations](/docs/traffic-management/transformations) | TrafficPolicy | 
 
-<!--
+## Policy merging {#policy-merging}
 
-## Policy inheritance rules when using route delegation
+{{< reuse "/docs/snippets/kgateway-capital.md" >}} lets you define how policies are merged when they are applied to a parent and child resource. 
 
-Policies that are defined in a TrafficPolicy resource and that are applied to a parent HTTPRoute resource are automatically inherited by all the child or grandchild HTTPRoutes along the route delegation chain. The following rules apply: 
+Parent-child hierarchies might be:
 
-* Only policies that are specified in a TrafficPolicy resource can be inherited by a child HTTPRoute. For inheritance to take effect, you must use the `spec.targetRefs` field in the TrafficPolicy resource to apply the TrafficPolicy resource to the parent HTTPRoute resource. Any child or grandchild HTTPRoute that the parent delegates traffic to inherits these policies. 
-* Child TrafficPolicy resources cannot override policies that are defined in a TrafficPolicy resource that is applied to a parent HTTPRoute. If the child HTTPRoute sets a policy that is already defined on the parent HTTPRoute, the setting on the parent HTTPRoute takes precedence and the setting on the child is ignored. For example, if the parent HTTPRoute defines a data loss prevention policy, the child HTTPRoute cannot change these settings or disable that policy.
-* Child HTTPRoutes can augment the inherited settings by defining TrafficPolicy fields that were not already set on the parent HTTPRoute. 
-* Policies are inherited along the complete delegation chain, with parent policies having a higher priority than their respective children.
+* Resources that target or serve other resources, such as Gateway > ListenerSet > HTTPRoute > Route rule.
+* Routes that are delegated, such as Parent HTTPRoute A > Child HTTPRoute B > Grandchild HTTPRoute C.
 
-For an example, see the [Policy inheritance](/docs/traffic-management/route-delegation/policy-inheritance/) guide.
+Policy merging applies to the following policies:
 
---> 
+* Native Kubernetes Gateway API policies, such as rewrites, timeouts, or retries.
+* {{< reuse "/docs/snippets/kgateway-capital.md" >}} TrafficPolicy.
```

Comment:
Would we ever reuse this index in the gg docs? i'm assuming no bc we cant render index page content... but maybe change the TrafficPolicy to the conref just in case

Reply:
actually wait, I suppose we could def create a non-index page in gg called like "about.md" and then just rebase this file onto that page